package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://github.com/joyent/node-qlocker#README.md",
   "dependencies": {
-    "fs-ext": "1.3.0"
+    "fs-ext": "git://github.com/joyent/node-fs-ext.git#node-0.10"
   },
```

Comment:
Would you object to adding `"engines" : { "node" : ">=0.10" },` to package.json to label intent that this maintain node 0.10 compatibility (or something similar in README.md perhaps)?

Reply:
Seems like I can't do that here with node 0.10

```
npm install
npm ERR! install Couldn't read dependencies
npm ERR! ReferenceError: nonStringDependency is not defined
```

But I think I should be doing this in the node-fs-ext branch anyway?

Reply:
Well, something on the node-qlocker repo would be theoretically helpful to maintainers (and less so to users) to know what version compatibility is required.

*Perhaps* for fs-ext, to avoid this warning during `npm install`:

```
npm WARN engine fs-ext@1.3.0: wanted: {"node":">= 8.0.0"} (current: {"node":"0.10.48","npm":"2.15.1"})
```

However, I could see the author rejecting that in a PR.


* * *

FWIW, I could not reproduce your `install Couldn't read dependencies` error:
https://gist.github.com/trentm/fd834eb291251c7a73f405bed8cd2bed

Reply:
Perhaps I couldn't repro because I got your fs-ext repo change.